pkgs/development/python-modules/openai-triton/default.nix

```diff
@@ -91,8 +91,7 @@ buildPythonPackage rec {
         --replace "include (\''${CMAKE_CURRENT_SOURCE_DIR}/googletest.cmake)" ""\
         --replace "include(GoogleTest)" "find_package(GTest REQUIRED)"
 
-      cat << \EOF > python/triton/common/build.py
-
```

Comment:
The linefeed was intentional but let's keep your current version so as not to restart ofborg

Reply:
ahh oops, sorry about that.